systemtest/playlist.xml

```diff
@@ -369,6 +371,65 @@
 		<groups>
 			<group>system</group>
 		</groups>
+		<impl>hotspot</impl>
+	</test>
+	
+	<test>
+		<testCaseName>LambdaLoadTest_OnenJ9_NonLinux</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>export JAVA_HOME=$(JDK_HOME)$(D)  && \
+	perl $(TEST_RESROOT)$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl  \
+	-test-root=$(Q)$(TEST_RESROOT)$(D)stf;$(TEST_RESROOT)$(D)openjdk-systemtest$(Q) \
+	-systemtest-prereqs=$(Q)$(TEST_RESROOT)$(D)systemtest_prereqs$(Q)  \
+	-java-args=$(Q)$(JVM_OPTIONS)$(Q) \
+	-results-root=$(REPORTDIR)  \
+	-test=LambdaLoadTest; \
+	$(TEST_STATUS)</command>
+		<subsets>
+			<subset>SE80</subset>
+			<subset>SE90</subset>
+			<subset>SE100</subset>
+		</subsets>
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>system</group>
+		</groups>
+		<impl>openj9</impl>
```

Comment:
`impl` has to be used under `impls`
```
		<impls>
			<impl>openj9</impl>
		</impls>
```

Same for the rest changes in the playlist.xml

Reply:
Fixed!